include/swift/Option/Options.td

```diff
@@ -1614,10 +1614,15 @@ def use_frontend_parseable_output:
 
 def print_explicit_dependency_graph:
   Flag<["-"], "print-explicit-dependency-graph">,
-  Flags<[HelpHidden, NewDriverOnlyOption]>,
+  Flags<[DoesNotAffectIncrementalBuild, HelpHidden, NewDriverOnlyOption]>,
   HelpText<"Print the result of module dependency scanning after external "
            "module resolution to output">;
 
+def explicit_dependency_graph_format:
```

Comment:
I imagine this shouldn't be interactive either, but we don't seem very consistent with applying that one/not 😅